lighthouse-core/gather/gatherers/start-url.js

```diff
@@ -11,20 +11,37 @@ const Gatherer = require('./gatherer.js');
 
 class StartUrl extends Gatherer {
   /**
-   * Grab the manifest, extract it's start_url, attempt to `fetch()` it while offline
+   * Go offline, assess the start url, go back online.
    * @param {LH.Gatherer.PassContext} passContext
    * @return {Promise<LH.Artifacts['StartUrl']>}
    */
   async afterPass(passContext) {
+    // Offline pass is not actually offline in `afterPass`,
+    // each gatherer must set and cleanup its own offline state.
```

Comment:
referencing the offline pass in the gatherers themselves makes me sad, though it's pretty necessary in `offline.js` at this point. For this one, though, would something like
``// `afterPass` is always online, so manually go offline to check start_url.`` or something like that work instead?

Reply:
sure, done!